### PR TITLE
feat: Log metrics summary in GitHub Action, RND-7

### PR DESCRIPTION
**Pull request by guilgaly**
Motivation:

We want to show the ongoing progress of the run

Modifications:

During the run, regularly retrieve and log metrics summary

### CODE REVIEW

File: src/run/metrics.ts [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  ]);
+
+  const currentTimestamp = Date.now();
+  const date = format(currentTimestamp, "yyyy-MM-dd HH:mm:ss");
+  const duration = formatDistanceStrict(currentTimestamp, runInfo.injectStart);
+
+  const nbUsers = seriesResponse
+    .map(({ values }) => {
+      return values.length === 0 ? 0.0 : values[values.length - 1];
+    })
```
Comment (Isammoc) [inline src/run/metrics.ts:25]:
```suggestion
    .map(({ values }) => values.length === 0 ? 0.0 : values[values.length - 1])
```

File: yarn-error.log [hunk 2]
```diff
@@ -0,0 +1,2304 @@
+Arguments:
```
Comment (Isammoc) [inline yarn-error.log:1]: I think that log should not be in commit :stuck_out_tongue:

Comment (guilgaly) [reply yarn-error.log:1]: fizzer1k-5597

Comment (guilgaly) [reply yarn-error.log:1]: Might as well add it to the gitignore file, while I'm at it.

File: .gitignore [hunk 3]
```diff
@@ -28,3 +28,6 @@ node_modules/
 
 # Compiled project files
 dist/
+
+# Yarn
+yarn-error.log
```
Comment (Isammoc) [inline .gitignore:33]: `*.log`?

**Review by Isammoc — APPROVED** [review]

**Pull request merged**